docs/guide/built-in-functions.md

```diff
@@ -226,7 +226,7 @@ Total number of functions: **{{ $page.functionsCount }}**
 | FORMULATEXT | Returns a formula in a given cell as a string.                                                                      | FORMULATEXT(Reference)                              |
 | HLOOKUP     | Searches horizontally with reference to adjacent cells to the bottom.                                               | HLOOKUP(Search_Criterion, Array, Index, Sort_Order) |
 | INDEX       | Returns the contents of a cell specified by row and column number. The column number is optional and defaults to 1. | INDEX(Range, Row [, Column])                        |
-| MATCH       | Returns the relative position of an item in an array that matches a specified value.                                | MATCH(Searchcriterion, Lookuparray, Type)           |
+| MATCH       | Returns the relative position of an item in an array that matches a specified value.                                | MATCH(Searchcriterion, Lookuparray, MatchType)      |
```

Comment:
This is unrelated to this task. See: https://github.com/handsontable/hyperformula/discussions/1148#discussioncomment-4759685

Reply:
This description follows the general assumption that _the functions work the same as in Microsoft Excel_. I don't like this assumption, but currently, [we do not have space for a more thorough documentation of the built-in functions](https://github.com/handsontable/hyperformula/issues/936).

So:
- MatchType accepts the same values as in [the Microsoft Excel implementation of `MATCH`](https://support.microsoft.com/en-us/office/match-function-e8dffd45-c762-47d6-bf89-533f4a37673a)
- it is indeed optional and I marked it as such in the documentation